lib/feature/class-feature.php

```diff
@@ -145,16 +145,31 @@ public static function is_enabled_by_percentage( string $feature ) {
 
 		// Which bucket is the site in - 100 possibilites, one for each percentage. We run this through crc32 with
 		// the feature name so that the same sites aren't always the canaries
-		$bucket = crc32( $feature . '-' . constant( 'FILES_CLIENT_SITE_ID' ) ) % 100;
+		$bucket = static::get_bucket( constant( 'FILES_CLIENT_SITE_ID' ), $feature );
 
 		// Is the bucket enabled?
 		$threshold = $percentage * 100; // $percentage is decimal
 
 		if ( is_multisite() && $bucket < $threshold ) {
```

Comment:
Should we move this logic inside `get_bucket` too? Since it's possible that `$bucket` will be smaller than `$threshold` and we need to check the crc32 against the blog ID

Reply:
No I don't think so, this would make the method [impure](https://en.wikipedia.org/wiki/Pure_function), which is something I don't like :)

Reply:
OK fair enough. I guess as long as one knows that doing `\Automattic\VIP\Feature::get_bucket( 31337, 'cool-feature' );` might not be the most accurate for multi-sites due to the added processing logic 🙂 since there are cases where it might use the blog ID instead of the app ID.

Reply:
@rebeccahum How would you phrase that comment?

Reply:
"Note: When using `get_bucket()` in a multisite, the blog ID may need to be used instead of app ID if `$threshold` exceeds the `$bucket`."